client/platform/web-girder/views/TrainedModels.vue

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
+import {
+  computed, defineComponent, onBeforeMount, ref,
+} from 'vue';
+import { usePrompt } from 'dive-common/vue-utilities/prompt-service';
+import { Pipelines, useApi, Pipe } from 'dive-common/apispec';
+import { DataTableHeader } from 'vuetify';
+import { useRouter } from 'vue-router/composables';
+
+export default defineComponent({
+  name: 'TrainedModels',
+  setup() {
+    const {
+      getPipelineList, deleteTrainedPipeline, exportTrainedPipeline,
+    } = useApi();
+    const { prompt } = usePrompt();
+    const router = useRouter();
+
+    const unsortedPipelines = ref({} as Pipelines);
+
+    onBeforeMount(async () => {
+      unsortedPipelines.value = await getPipelineList();
+    });
+
+    const trainedModels = computed(() => {
+      if (unsortedPipelines.value.trained) {
+        return unsortedPipelines.value.trained.pipes;
+      }
+      return [];
+    });
+
+    async function deleteModel(item: Pipe) {
+      const confirmDelete = await prompt({
+        title: `Delete "${item.name}" model`,
+        text: 'Are you sure you want to delete this model?',
+        positiveButton: 'Delete',
+        negativeButton: 'Cancel',
+        confirm: true,
+      });
+
+      if (confirmDelete) {
+        try {
+          await deleteTrainedPipeline(item);
+          unsortedPipelines.value = await getPipelineList();
+        } catch (err) {
+          let text = 'Unable to delete model';
+          if (err.response?.status === 403) text = 'You do not have permission to run training on the selected resource(s).';
+          prompt({
+            title: 'Delete Failed',
+            text,
+            positiveButton: 'OK',
+          });
+        }
+      }
+    }
+
+    async function exportModel(item: Pipe) {
+      await exportTrainedPipeline(item.folderId!, item);
+      router.push('/jobs');
+    }
+
+    const trainedHeadersTmpl: DataTableHeader[] = [
+      {
+        text: 'Model',
+        value: 'name',
+        sortable: true,
+      },
+      {
+        text: 'Export',
+        value: 'export',
+        sortable: false,
+        width: 80,
+      }, {
+        text: 'Delete',
+        value: 'delete',
+        sortable: false,
+        width: 80,
+      },
+    ];
+
+    return {
+      deleteModel,
+      exportModel,
+      items: trainedModels,
+      headers: trainedHeadersTmpl,
+    };
+  },
+});
+</script>
+
+<template>
+  <v-container :fluid="$vuetify.breakpoint.mdAndDown">
+    <v-card class="trained-models-wrapper mt-4 pa-6">
```

Comment:
Maybe an explanation of what these models are in a <p> or something in the begigging..  They aren't just generic 'models' they are user-trained models shared with the current user.  I'm just suggesting this so I don't get a ton of emails asking why the default VIAME models aren't in here.